guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

```diff
@@ -16,22 +16,19 @@ Ensure that you have enough space for the migration.
 A rough estimation of the space you require is 2-3 times the size of the MongoDB Pulp 2 database.
 
 .Pre-Migration Checks
+The sequence should always be if you have any Composite Content Views, take action on those first, and then work on the Content Views.
+
 * Ensure that all enabled repositories complete the synchronization process.
 * If any repositories are synced with a *Warning* or *Error* state, fix the underlying problem and resync the repository.
 * Ensure that no paused tasks in the system are related to any repositories or Content Views.
 * If you no longer require specific Content Views or Composite Content View Versions, you can perform a cleanup action.
 * For more information about how to perform the cleanup, see https://access.redhat.com/solutions/2760531[How to remove old content view versions in Red Hat Satellite 6 using CLI/hammer?] .
-* If some Content View or Composite Content View versions are not required anymore, please consider performing a cleanup of them. https://access.redhat.com/solutions/2760531[How to remove old content view versions in Red Hat Satellite 6 using CLI/hammer?] can be referred to understand how to perform the cleanup of older Content View/Composite Content View versions using `hammer_cli`.
-
-[NOTE]
-====
-The sequence should always be if you have any Composite Content Views, take action on those first, and then work on the Content Views.
-====
-
+* If some Content View or Composite Content View versions are not required anymore, please consider performing a cleanup of them.
+* For more information about how to perform the cleanup of older Content View/Composite Content View versions using `hammer_cli`, see https://access.redhat.com/solutions/2760531[How to remove old content view versions in Red Hat Satellite 6 using CLI/hammer?].
```

Comment:
Please do not use slashes. I mentioned this in my first review. 

Content View and Composite Content View Versions.